engine/launcher/src/main/scala/org/enso/launcher/cli/LauncherApplication.scala

```diff
@@ -112,11 +112,12 @@ object LauncherApplication {
       "jvm",
       "These parameters will be passed to the launched JVM as -DKEY=VALUE."
     )
-  private def jvmMode =
-    Opts.flag(
+  private def jvm =
+    Opts.optionalParameter[Path](
```

Comment:
- I have to admit I am not sure at all how to represent this **optional option with optional parameter** in this system?
- in 8728e1a I at least fixed the compilation
- but using `Option(jvm)` at various places is unlikely correct, right @radeusgd?